hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/AbstractHtsCall.java

```diff
@@ -82,6 +82,15 @@ protected PricedResult completionWith(
         return gasOnly(successResult(output, gasRequirement, recordBuilder), recordBuilder.status(), isViewCall);
     }
 
+    protected PricedResult completionWith(
```

Comment:
Is this method used anywhere?

Reply:
This was a left-over from before reverting some of the changes in this PR. It's removed now.